docs/tempo/website/configuration/_index.md

```diff
@@ -634,3 +635,67 @@ memberlist:
     [packet_write_timeout: <duration> | default = 5s]
 
 ```
+
+## Overrides
+Tempo provides a overrides module for user to set global or per-tenant override settings.   
+**Currenly only ingestion limits can be overrided.**
+
+### Ingestion limits
+The default limits in Tempo may not be sufficient in high volume tracing environments. 
+```
+# Overrides configuration
+overrides:
+    
+    # Global ingestion limits configurations
+
+    # Burst size (bytes) used in ingestion.
+    # (Default: `20,000,000` ~20MB )
+    [ingestion_burst_size_bytes: <int>]     # RATE_LIMITED Error when exceeded
```

Comment:
> RATE_LIMITED Error when exceeded

Can we add these as a comment above the config option with an example like in the other doc? I.e.. like this:

```
# Results in errors like
#    RATE_LIMITED: ingestion rate limit (15000000 bytes) exceeded while adding 10 bytes
```